jobs/grafana/templates/credentials

```diff
@@ -0,0 +1,4 @@
+[default]
+aws_access_key_id = <%= p("grafana.aws.aws_access_key_id") %>
```

Comment:
Empty string should be fine as long as it parses. Not sure if it does in this format.